src/Apps/Settings/Routes/EditProfile/Components/SettingsEditProfileFields.tsx

```diff
@@ -228,13 +229,14 @@ const SettingsEditProfileFields: React.FC<SettingsEditProfileFieldsProps> = ({
                 data-testid="edit-profile-other-relevant-positions-input"
               />
 
-              <Input
+              <TextArea
                 title="About"
                 placeholder="Add a brief bio, so galleries know which artists or genres you collect."
                 name="bio"
-                multiple
                 maxLength={150}
-                onChange={handleChange}
+                onChange={({ value }) => {
+                  setFieldValue("bio", value)
+                }}
```

Comment:
**Q:** `onChange={handleChange}` doesn't work with `TextArea` component?

Reply:
Unfortunately, it doesn't 😅 I tried!

Reply:
Probably because Formik [expects an event](https://github.com/artsy/palette/blob/main/packages/palette/src/elements/TextArea/TextArea.tsx#L25) but `TextAreay` passes [TextAreaChanged](https://github.com/artsy/palette/blob/main/packages/palette/src/elements/TextArea/TextArea.tsx#L25).